hs_core/models.py

```diff
@@ -4132,12 +4132,10 @@ def create_date_node(date, date_type):
         dataset_licenses_node = etree.SubElement(dataset_node, '{%s}program' % ai, name="AccessIndicators")
```

Comment:
We probably don't need to even create the `dataset_license_node` if we first check `rights.url` exists

Reply:
good point thanks, resolved that